server/src/test/java/com/linecorp/centraldogma/server/internal/api/TokenServiceTest.java

```diff
@@ -13,6 +13,7 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
+
```

Comment:
nit: Revert? 😃

Reply:
I wasn't sure which is the preferred way, since I've seen both... Let me revert!